docs/config.yaml

```diff
@@ -255,6 +255,20 @@ authenticators:
     # Sets the anonymous username. Defaults to "anonymous". Common names include "guest", "anon", "anonymous", "unknown".
     subject: anonymous
 
+  # Configures the cookie session authenticator
+  cookie_session:
+    # Set enabled to true if the authenticator should be enabled and false to disable the authenticator. Defaults to false.
+    enabled: true
+
+    # Sets the origin to proxy requests to. If the session store returns a 200 with body `{ "subject": "...", "extra": {} }`
+    # The request will pass the subject through successfully, otherwise it will be marked as unauthorized
+    session_store_origin: https://origin-of-session-store
```

Comment:
To stay consistent with other configs, how about `session_store_url`? Or maybe `check_session_url`?

Reply:
Sounds good, while this is technically an origin since it's protocol + host but no path, I guess the other "URL"s are too. Consistency is probably better here. I like `check_session_url`!